packit.yaml

```diff
@@ -3,7 +3,8 @@ actions:
   create-archive:
     # the upstream git spec file has unresolved macros which are normally put
     # in at `make dist` time; they are not important for CI
-    - sh -c 'sed "s/%{npm-version:.*}/0/" tools/cockpit.spec > cockpit.spec'
+    # also, build optional packages for CentOS 8
```

Comment:
Heh, was wondering how would you convince that logic to build all of it - now I see :)